src/tests/Interop/NativeLibrary/Callback/CallbackStressTest_TargetWindows.csproj

```diff
@@ -3,7 +3,7 @@
     <!-- Needed for CLRTestTargetUnsupported, CMakeProjectReference -->
     <RequiresProcessIsolation>true</RequiresProcessIsolation>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <DefineConstants >WINDOWS</DefineConstants>
+    <DefineConstants>$(DefineConstants);WINDOWS</DefineConstants>
```

Comment:
Note: this is not redundant. `WINDOWS` is only defined when OS-specific TFM is used https://learn.microsoft.com/dotnet/standard/frameworks#preprocessor-symbols (which in this is not `net9.0` vs. `net9.0-windows`)